app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -94,18 +93,32 @@ def destroy
     end
   end
 
+  def sublayout
+    'api/service' if @api_docs_service && service
+  end
+
   private
 
+  def service
+    @service ||= @api_docs_service.service
+  end
+
   def api_docs_params(*extra_params)
     permit_params = %i[name body description published skip_swagger_validations service_id] + extra_params
     params.require(:api_docs_service).permit(*permit_params)
   end
 
+  def activate_service_navigation
+    # TODO: this next line is a copy-paste from Api::BaseController :(
+    activate_menu submenu: current_account.multiservice? ? :services : @service.name
```

Comment:
just trying to keep the same behaviour for all submenus under service 😄 
https://github.com/3scale/porta/blob/56866b20ea29e028ce18389be575481e6cccb656/app/controllers/api/base_controller.rb#L8-L10

I don't know why, but either we do it that way always or never I guess 😄

Reply:
It should work with `activate_menu submenu: : @service.name` no?

Reply:
What you quote is for the Base in general but here you override it.
BTW, activate_submenu is deprecated

Reply:
I will figure it out 🤔

Reply:
I am not overridding anything because it is not inheriting from that controller 😝 That's why I said I don't like that copy-paste but it is the only behavior I want from that other controller lol

Reply:
> BTW, activate_submenu is deprecated

Well, that's new info 😝 That is not in an comment in there and it should be... and it is being used for everything under the service submenu 🤔 and if it is deprecated, what should we use instead? @hallelujah

Reply:
https://github.com/3scale/porta/blob/master/lib/menu_system.rb#L127-L131

Reply:
That's a completely different `activate_submenu` 😂 with a completely different code and behaviour 😂 Check the one I sent you instead 😝

Reply:
For example, one of the many usages of the method I am talking about:
https://github.com/3scale/porta/blob/56866b20ea29e028ce18389be575481e6cccb656/app/controllers/api/applications_controller.rb#L9

Reply:
Yes so the method is overriden.

My point is to only do `activate_menu submenu: @service.name`
It will override `active_menus[:submenu]` with the value `@service.name`

Reply:
@hallelujah 

What you are suggesting is:
`activate_menu  submenu:  @service.name`
And what I am doing:
`activate_menu submenu: current_account.multiservice? ? :services : @service.name`

The reason why I did this is because all usages of any controller using the service submenu is doing it that way (like the applications_controller I've just shown you) and I wanted to keep the same behaviour for all.
If it is necessary, I don't know, I am figuring that out for now... 😄